### DISCUSSION TITLE
Cr 10 Smart Pro

### DISCUSSION THREAD

**Discussion by Sici29** [CrealityOfficial/Creality_Sonic_Pad#2] [General]
Hi everyone, I have reached point 29 of the guide but now I need the following information:

Micro-controller Architecture 、 Processor model 、 Bootloader Offset 、 Clock Reference 、 Communication interface etc. parameters

to configure the firmware of the CR 10 Smart Pro, can someone help me?

**Comment by Str8Gamin420** [comment]
we are on our own... looking at the response rates, practiculy never, Creality should be more involved, especially when it comes to customer relations.
